test/intl402/Temporal/Calendar/prototype/monthDayFromFields/one-of-era-erayear-undefined.js

```diff
@@ -1,19 +0,0 @@
-// Copyright (C) 2023 Igalia, S.L. All rights reserved.
```

Comment:
Renamed to `fields-underspecified.js` because I added more cases that were not specifically about era/eraYear.